armory/metrics/task.py

```diff
@@ -1901,3 +1956,47 @@ def __call__(self, y_list, y_pred_list):
                     results[f"mean_{k.lower()}"] = value
 
         return results
+
+
+populationwise(GlobalHOTA, name="hota_metrics")
+
+
+@result_formatter("total_wer")
+def total_wer_formatter(result):
+    total, (num, denom) = result
+    return f"total={float(total):.3}, {num}/{denom}"
+
+
+@result_formatter("word_error_rate")
+def word_error_rate_formatter(result):
+    result = total_wer(result)
+    return total_wer_formatter(result)
+
+
+result_formatter("total_word_error_rate")(total_wer_formatter)
```

Comment:
Is there a reason to be able to refer to `total_wer_formatter` in `FORMATTERS` with two different keys?

Reply:
I meant it for backwards compatibility. But it's no longer needed, as it was only used in the old `instrument.config`